src/game/Weather.h

```diff
@@ -47,7 +47,8 @@ enum WeatherState
     WEATHER_STATE_HEAVY_SNOW        = 8,
     WEATHER_STATE_LIGHT_SANDSTORM   = 22,
     WEATHER_STATE_MEDIUM_SANDSTORM  = 41,
-    WEATHER_STATE_HEAVY_SANDSTORM   = 42
+    WEATHER_STATE_HEAVY_SANDSTORM   = 42,
+    WEATHER_STATE_SCOURGE_INVASION  = 86 // [Event: Scourge Invasion] Weather
```

Comment:
If you changed this based on the value in the sniff, you should know the states in the new client are different than those in the old one. The replay core reinterprets the weather state.

https://github.com/ratkosrb/core/blob/sniffviewer/src/game/Replay/SniffedEvent.cpp#L171

Reply:
means I can theoretically undo all changes in weather.h and weather.cpp and use just:
me->GetMap()->SetWeather(zoneid, WEATHER_TYPE_STORM, 0.25f, true);